Go_Refined_Code/database/db_migrate.py

```
@@ -0,0 +1,123 @@
+pg_cursor = pg_conn.cursor()
+
+# --- Get all tables ---
+mysql_cursor.execute("SHOW TABLES")
+tables = [list(row.values())[0] for row in mysql_cursor.fetchall()]
+print(f"Found tables: {tables}\n")
+
+for table in tables:
+    print(f"Migrating table: {table}")
+
+    # Get total row count for progress tracking
+    mysql_cursor.execute(f"SELECT COUNT(*) as cnt FROM {table}")
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)